libs/feature/editor/src/lib/components/record-form/form-field/form-field-rich/form-field-rich.component.html

```diff
@@ -12,10 +12,10 @@
     <span
       class="material-symbols-outlined gn-ui-icon-small"
       [ngClass]="{ 'text-primary': !preview }"
-      >{{ preview ? 'visibility' : 'visibility_off' }}</span
+      >{{ preview ? 'visibility_off' : 'visibility' }}</span
```

Comment:
is that intentional? I haven't noticed that this was inverted

Reply:
It is intentional. We kind of got used to it this way, but I felt it was wrong the first time I coded it. And now with François fresh view on it, it felt odd too.